Doc/whatsnew/3.8.rst

```diff
@@ -375,6 +375,11 @@ Optimizations
   This makes the created list 12% smaller on average. (Contributed by
   Raymond Hettinger and Pablo Galindo in :issue:`33234`.)
 
+* Doubled the speed of class variable writes.  When a non-dunder attribute
+  was updated, there was an unnecessary call to update slots.
+  (Contributed by Stefan Behnel, Pablo Galindo Salgado, Raymond Hettinger,
```

Comment:
No need to mentioned me here. I just left few review comments. The author of the idea and the implementation is Stefan.

Reply:
I'm happy with having all the names in there. It was a joined effort, with lots of good reviews.